### PR TITLE
[PIPE-322] Few changes to allow copy back of Recipient Lookup

### PR DESCRIPTION
**Pull request by sethstoudenmier**
**Description:**
Few changes that allow the copy back of Recipient Lookup to 

**Technical details:**
* docker-compose.yaml was updated to point to Postgres 13 image to allow use of `INCLUDING GENERATED` in the loader
* `INCLUDING GENERATED` was added to copy back references to the sequence
* `load_table_from_delta` was updated to optionally reset a sequence; resets the sequence back to previous value if the job fails
* updated `travis.yml` to use Postgres 13 as well

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
4. [x] Matview impact assessment completed
5. [x] Frontend impact assessment completed
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created
8. [ ] Jira Ticket [PIPE-322](https://federal-spending-transparency.atlassian.net/browse/PIPE-322):
    - [x] Link to this Pull-Request
    - [ ] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/etl/management/commands/copy_table_metadata.py
Comment (kwhickey) [inline usaspending_api/etl/management/commands/copy_table_metadata.py:17]: I felt like we went through a discussion on this with transaction_search and award_search, and decided to NOT copy unique constraints.

Or ... did we just decide to not copy them for _those_ tables, because they weren't actually necessary on those tables. So we just kept on _not copying them_? 

If the latter, what is it about the unique constraints you're seeing that makes you feel like we ought to keep them, rather than discard them like we did for award_search and transaction_search?

File: usaspending_api/recipient/delta_models/recipient_lookup.py
Comment (kwhickey) [inline usaspending_api/recipient/delta_models/recipient_lookup.py:29]: When did the postgres columns transition over to the column set that was _without ID_? I thought the idea was we were (maybe not permanently) keeping ID in there and letting it auto-populate with the sequence?

File: usaspending_api/etl/management/commands/load_table_from_delta.py
Comment (collinwr) [inline usaspending_api/etl/management/commands/load_table_from_delta.py:111]: Is this second line of the parameter help info directly related to the parameter? Looks like the sequence value is reset to the previous value whether this argument is used or not.

File: usaspending_api/etl/management/commands/load_table_from_delta.py
Comment (kwhickey) [inline usaspending_api/etl/management/commands/load_table_from_delta.py:342]: Is the plan still to see if we can drop this as technical debt, by dropping the PK column from the table so that no sequence needs to be maintained? 
What are the remaining challenges to doing so? Django having a PK?

**Review by kwhickey — COMMENTED** [review]
Some questions for clarification came up in the review.

File: usaspending_api/recipient/delta_models/recipient_lookup.py
Comment (sethstoudenmier) [reply usaspending_api/recipient/delta_models/recipient_lookup.py:29]: The table is created via the `LIKE` command which created the `id` column that references the sequence. This statement is all of the columns we are looking to copy back from Delta to Postgres. Not having `id` specified means that it will be auto populated by the sequence.

File: usaspending_api/etl/management/commands/load_table_from_delta.py
Comment (sethstoudenmier) [reply usaspending_api/etl/management/commands/load_table_from_delta.py:342]: >Is the plan still to see if we can drop this as technical debt, by dropping the PK column from the table so that no sequence needs to be maintained?
Not entirely sure if that is planned technical debt or not.

>What are the remaining challenges to doing so? Django having a PK?
Most likely it is just that, but we could trick Django into thinking a PK column exists. The problem is that I have no idea what issues that could cause if Django expects something like a PK but doesn't have one.

File: usaspending_api/etl/management/commands/load_table_from_delta.py
Comment (sethstoudenmier) [reply usaspending_api/etl/management/commands/load_table_from_delta.py:111]: Sequence value is only reset if there is a value for `postgres_seq_last_value`; which would only exist if it was previously changed.

Comment (collinwr) [reply usaspending_api/etl/management/commands/load_table_from_delta.py:111]: Okay gotcha. I'm following

**Review by kwhickey — DISMISSED** [review]
Questions answered, and approving for now and can again once caught up with merges.

**Review by collinwr — APPROVED** [review]

**Pull request merged**
